specification/batch/data-plane/Microsoft.Batch/stable/2024-02-01.19.0/BatchService.json

```diff
@@ -90,7 +90,9 @@
             "type": "integer",
             "format": "int32",
             "default": 30,
-            "description": "The maximum time that the server can spend processing the request, in seconds. The default is 30 seconds.",
+            "minimum": 1,
+            "maximum": 30,
+            "description": "The maximum time that the server can spend processing the request, in seconds. Defaults to the maximum value of 30 seconds.",
```

Comment:
Since TLSFE actually does't reject request with timeout > 30, should we not define "maximum", instead just update the description to like: "The default is 30 seconds. If specified value is larger than 30 seconds, the server will ignore it and still use the default value."

Reply:
Good point - I'll update the PR

Reply:
I tried to shorten the text a little bit, but keep the spirit of what you suggested.